### DISCUSSION TITLE
Difficulties with "play <example artist> in Amperfy"

### DISCUSSION THREAD

**Discussion by kaytwo** [BLeeEZ/amperfy#223] [Q&A] [answered]
Great app! I've been trying to get the siri integration feature to work (specifically with a HomePod, but also on device), but am having two main problems:

* Siri always interprets me saying "Amperfy" as "Amplify" - even if I try saying "Amperfy Music" it is still recognized as "amplify music."
* If I get around that using type to siri or the "create a contact with the name Amperfy and set a custom pronunciation" trick, I still get various errors, either "You'll need to continue in Amperfy":
![IMG_4600](https://github.com/BLeeEZ/amperfy/assets/587187/ae2af859-39fb-42c4-94d8-937868d0583c)
or "I don't see an app for that, you'll need to download one."
![IMG_4602](https://github.com/BLeeEZ/amperfy/assets/587187/94ecbfdb-58d7-4425-b898-c7286440b874)

Has anyone figured out any tricks/workarounds to get the "play in" functionality working? Thanks!

**Comment by BLeeEZ** [comment, marked as answer]
I can use Siri without saying Amperfy. When I say "Play AC/DC" it asks me the first time which App should be used. All following commands ask then directly send to Amperfy.

**Reply by kaytwo** [reply]
Thanks! The next day without making any changes to my setup, things are working just fine. Maybe Siri just needed to learn what I was trying to say.

Even if I do this command from the HomePod, it plays by default on the iPhone which is awkward and then hard to transfer to the HomePod, but that might be a Siri issue / beyond Amperfy's control. Thanks again loving the app!
